pkg/cmd/pr/create/create.go

```diff
@@ -28,6 +28,11 @@ type browser interface {
 	Browse(string) error
 }
 
+//go:generate mockery --name gitClient --structname GitClient
```

Comment:
Running `go generate ./...` in our project will now update the associated mocks after we edit an interface.